test_driver/auth_test.dart

```diff
@@ -0,0 +1,90 @@
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+void main() {
+  group('Auth section', () {
+    // create finders
+    final emailAuthButton = find.byType('EmailOptionsFAB');
```

Comment:
felt kind of messy to create so many finders up the top, open to suggestions for cleaning her up or leaving if shes alg. I can see the finders list growing alot with a bigger integration test, but maybe worry about that potential clutter when it comes?

Reply:
Yeah I feel you - the only thing off the top of my head I can think of would be to wrap them in a class or just as top level variables in a separate file.  I agree though, worry about it when it cones :-)